src/nupic/utils/LoggingException.cpp

```diff
@@ -34,7 +34,11 @@ LoggingException::~LoggingException() throw()
   if (!alreadyLogged_) {
     // Let LogItem do the work for us. This code is a bit complex
     // because LogItem was designed to be used from a logging macro
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
+    auto li = new LogItem(filename_.c_str(), lineno_, LogItem::info);
```

Comment:
> I don't fully understand your proposal but if you plan to leave some work for future PRs then please file an issue for that work and put a comment here with the link so we don't lose track.

Is there a strong reason to avoid emit `info` instead of `error`? If not, we could use only `info` and avoid use `#if defined(NTA_OS_WINDOWS)`.